mautic/mautic
# bugfix - add back variant button on email dyn content

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/4089
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Add back the button "new" bellow and make is works
![image](https://user-images.githubusercontent.com/14074272/27951039-d66bef38-6303-11e7-9085-5cfdc6e967f6.png)


#### Steps to test this PR:
1. Apply the PR
2. Make dyn content with variant and save it.
3. Come back and check the content is here
